src/web/src/styles/telescope-post-content.css

```diff
@@ -104,7 +118,7 @@
 .telescope-post-content code,
 .telescope-post-content pre {
   overflow-wrap: unset;
-  word-wrap: none;
+  word-wrap: unset;
```

Comment:
Would it be nice to make the cursor a zoom icon to indicate an image is zoomable? Maybe a zoom-out icon if the image has already zoomed.

Reply:
added in the updated css along with background blur and color on zoom